test/absinthe_ecto_test.exs

```diff
@@ -1,8 +1,34 @@
-defmodule AbsintheEctoTest do
-  use ExUnit.Case
-  doctest AbsintheEcto
+defmodule Absinthe.EctoTest do
+  use Absinthe.Ecto.EctoCase
+  import Absinthe.Ecto
+  doctest Absinthe.Ecto
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  describe "ecto_batch/3" do
+    test "it returns batch middleware that performs a valid Ecto query" do
+      user = insert(:user)
+      post = insert(:post, user: user) |> Absinthe.Ecto.TestRepo.reload
```

Comment:
Since the user is initially loaded here, we reload the record so that we can use it to assert on the return value of `perform_batch` which doesn't preload the post's user.